test/OWNERS.yaml

```diff
@@ -1,5 +1,17 @@
 # For an explanation of the OWNERS.yaml rules and syntax, see:
 # https://github.com/ampproject/amp-github-apps/blob/master/owners/OWNERS.example.yaml
 
-- ampproject/runtime
-- ampproject/validator
+- ampproject/wg-access-subscriptions
+- ampproject/wg-ads
+- ampproject/wg-amp4email
+- ampproject/wg-analytics
+- ampproject/wg-approvers
+- ampproject/wg-caching
+- ampproject/wg-codeofconduct
+- ampproject/wg-infra
+- ampproject/wg-outreach
+- ampproject/wg-performance
+- ampproject/wg-runtime
+- ampproject/wg-stories
+- ampproject/wg-ui-and-a11y
+- ampproject/wg-viewers
```

Comment:
Was `wg-validator` left out on purpose? (It could well be that this makes sense.)

Reply:
Nope. For some reason they are not on this page,  https://github.com/orgs/ampproject/teams/working-groups/teams

Reply:
I think their group is covered by `wg-caching` @Gregable do you want to specifically add `wg-validator` as well?

Reply:
I added `ampproject/wg-caching` to the root OWNERS file for validator files, as well as the validator directory, so I'd guess that's probably what you're looking for. I think `dvoytenko` was the one to direct that, lemme confirm

Reply:
Correction; after double-checking in https://github.com/ampproject/amp-github-apps/issues/395 , I see that it was @honeybadgerdontcare . Perhaps you could weigh in?

Reply:
Also, if wg-validator is not a valid team, please do not add it :grimacing:

Reply:
I was wrong about `wg-validator` being the Conduit working group's name. `wg-caching` is correct.